Libraries/src/Amazon.Lambda.AspNetCoreServer.Hosting/Internal/LambdaRuntimeSupportServer.cs

```diff
@@ -87,9 +87,28 @@ public class APIGatewayHttpApiV2MinimalApi : APIGatewayHttpApiV2ProxyFunction
             /// Create instances
             /// </summary>
             /// <param name="serviceProvider">The IServiceProvider created for the ASP.NET Core application</param>
-            public APIGatewayHttpApiV2MinimalApi(IServiceProvider serviceProvider)
+            /// <param name="encodingOptions">The encoding options to register</param>
+            public APIGatewayHttpApiV2MinimalApi(IServiceProvider serviceProvider, IEncodingOptions? encodingOptions = null)
                 : base(serviceProvider)
             {
+                if(encodingOptions != null)
```

Comment:
This code block and the one under method APIGatewayRestApiMinimalApi are identical. Can you create a private method and call it from both methods?